_data/projects/30SecondsOfC++.yml

```diff
@@ -1,5 +1,7 @@
+---
```

Comment:
This document separator is used in a few ways by YAML:

 - to prefix a version at the start of the file
 - to store several documents in the one file

Neither of these are used by us, but I couldn't find an option in [psych](https://github.com/ruby/psych) to turn this off.